app/unitPages.rb

```diff
@@ -637,7 +702,7 @@ def getUnitSidebarWidget(unit, widgetID)
 def travNav(navBar, &block)
   navBar.each { |nav|
     block.yield(nav)
-    if nav['type'] == 'folder'
+    if nav['type'] and nav['type'].include?('folder')
```

Comment:
Works fine, but in general "and" in this context should be "&&"; I think we conventionally save "and" for an expression on the left that conditionally triggers a command on the right. But please correct me if I'm off-base here.